server/services/operations_service.go

```diff
@@ -125,18 +128,70 @@ func searchFilterFunc(query string) func(b *pb.Blurb) bool {
 	}
 }
 
+// returns a successful response if the resource name is not blank
```

Comment:
A godoc comment must begin with the method name

`// CancelOperation ...`

Reply:
TIL - thanks! done.